service/src/test/java/uk/nhs/adaptors/gp2gp/ehr/mapper/ObservationStatementMapperTest.java

```diff
@@ -72,6 +78,13 @@
         + "expected-output-observation-statement-9.xml";
     private static final String OUTPUT_XML_WITH_REFERENCE_RANGE = TEST_FILE_DIRECTORY
         + "expected-output-observation-statement-10.xml";
+    private static final String OUTPUT_XML_WITH_INTERPRETATION_CODE = TEST_FILE_DIRECTORY
+        + "expected-output-observation-statement-11.xml";
+    private static final String OUTPUT_XML_WITH_INTERPRETATION_CODE_NO_SYSTEM = TEST_FILE_DIRECTORY
+        + "expected-output-observation-statement-12.xml";
+    private static final String OUTPUT_XML_WITH_INTERPRETATION_CODE_INVALID_CODE = TEST_FILE_DIRECTORY
```

Comment:
```suggestion
    private static final String OUTPUT_XML_WITH_INTERPRETATION_INVALID_CODE = TEST_FILE_DIRECTORY
```
Will it be a better name? Also will be consistent in this case with the INPUT_JSON_WITH_INTERPRETATION_INVALID_CODE.
What do you think?

Reply:
I've renamed the variable name to better reflect that is being tested in fdc482d